README.md

```diff
@@ -192,6 +192,76 @@ In order to package libraries when connected to an Orchestrator instance, ensure
 
 Do you have any questions regarding the plugin? Ask them [here](https://connect.uipath.com/marketplace/components/jenkins-plugin-for-uipath-public-preview/questions).
 
+## Troubleshooting
+#### [Unauthorized error](#markdown-unauthorized-error)
+#### [Forbidden error](#markdown-forbidden-error)
+#### [Folder/environment not found](#markdown-folder-not-found)
+#### [Package already exists (Conflict)](#markdown-package-already-exists)
+#### [Workspace in Jenkins default installation location](#markdown-workspace-not-accessible)
+#### [Failed to deploy a package which contains non-Latin characters in project name / path to project](#markdown-specify-file-encoding)
+
+### Unauthorized error
+If using basic authentication:
+* ensure the correctness of the username-password combination on the web login
+* if federated authentication is enabled, make sure your write the username in the task as “DOMAIN\user”
+
+If using token authentication:
+* For automating staging.uipath.com or alpha.uipath.com, please reach out to Alvin Stanescu, as some additional configuration might be required
+* Regenerate the token and use new credentials
+
+If authenticating against on an on-premise Orchestrator you might receive this error as a result of the certificate used for the Orchestrator not being valid. This might mean that it has the wrong CN or other validation issues. Please check the certificate if the above mentioned are correct.
+
+### Forbidden error
+Likely, the user does not have the permission to perform the action.
+
+Ensure that the user has permissions to read folders, upload packages, create and update processes, read test sets, test cases, create test sets and run test sets.
+
+### Folder/environment not found
+Ensure that the authenticated user used by CI/CD plugins has the Folders.View and (20.4 only) BackgroundTask.View permissions.
+
+### Package already exists (Conflict)
+Ensure that the package that you are trying to deploy does not exist with the same version already. If it does, consider using automatic package versioning, so that the new version is bumped up every time we deploy.
+
+### Jenkins workspace is in a protected path
+Build may fail if the Jenkins workspace is inside a location on disk (like C:\Windows or C:\Program Files) to which the user does not have permissions.
```

Comment:
I would change this to something along the lines of (since the user might not realize that they have an issue with the workspace being in a protected path):

### Failed to run the command (generic error)

If the Jenkins workspace is inside a location on disk (like `C:\Windows` or `C:\Program Files`) to which the user does not have permissions, ensure that the workspace is placed on a path that can be accessed smoothly by the user